File: Modul-3/Soal-2/PRAK302-2410817210027-HAFIZPERDANA.py
nilai = int(input(""))

if nilai > 100 or nilai < 0:
    print("Error")
elif nilai >= 80:
    print("A")
elif nilai >= 70:
    print("B")
elif nilai >= 60:
    print("C")
elif nilai >= 50:
    print("D")
else:
    print("E")

File: Modul-3/Soal-2/PRAK302-2410817210027-HAFIZPERDANA.c
#include <stdio.h>
int main() {
    int nilai;

    printf("");
    scanf("%d", &nilai);
    
    if (nilai > 100) {
        printf("Error");
    } else if (nilai >= 80) {
        printf("A");
    } else if (nilai >= 70) {
        printf("B");
    } else if (nilai >= 60) {
        printf("C");
    } else if (nilai >= 50) {
        printf("D");
    } else {
        printf("E");
    }
    
    return 0;
}